.github/workflows/publish.yml

```diff
@@ -15,9 +15,9 @@ jobs:
       with:
         python-version: 3.x
     - name: Install dependencies
-      run: pip install build -e .
+      run: pip install flit
     - name: Create packages
-      run: python -m build -n -s -w .
+      run: flit build --setup-py
```

Comment:
You can't get `python -m build --sdist` to include a setup.py (it's not part of flit-core). That's a feature of `flit build --setup-py`. Not fond of injecting backend features via a frontend, but that's what Flit does. As long as `flit` is used to make the SDist for uploading to PyPI, it can include a setup.py. I just was making sure (in that commit) that using standard tools included everything else correctly.

Reply:
(If you don't care about the compatibly setup.py, then we can stick to standard tools and ignore the flit CLI)

PS: Sorry for spamming GitHub notifications, should have used a review to respond...

Reply:
I definitely want to include a compatibility setup.py in the 0.x series, but drop it in 1.0.